ProcessLib/LiquidFlow/LiquidFlowLocalAssembler-impl.h

```diff
@@ -76,35 +128,111 @@ void LiquidFlowLocalAssembler<ShapeFunction, IntegrationMethod, GlobalDim>::
     //       the integration loop for non-constant porosity and storage models.
     double porosity_variable = 0.;
     double storage_variable = 0.;
+    const double temperature =
+        MaterialLib::PhysicalConstant::CelsiusZeroInKelvin + 18.0;
```

Comment:
After PR #1722  merged, there will be a PR about reading the reference temperature from project file.